From f45e22e0150e68a7bdaba34574e60e1ce168fb7e Mon Sep 17 00:00:00 2001
From: Bartlomiej Horiszny <bartlomiejx.horiszny@intel.com>
Date: Mon, 13 May 2019 14:19:48 +0200
Subject: [PATCH] ASoC: Intel: Skylake: NHLT log level change

Change of the log level of querying NHLT blob from
warning to debug. Warning level was misinforming.

Change-Id: I1afbad3c2770f63ba1e7959c23e3d360faee6b97
Signed-off-by: Bartlomiej Horiszny <bartlomiejx.horiszny@intel.com>
---
 sound/soc/intel/skylake/skl-nhlt.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/intel/skylake/skl-nhlt.c b/sound/soc/intel/skylake/skl-nhlt.c
index 9142b06..5d65c86 100644
--- a/sound/soc/intel/skylake/skl-nhlt.c
+++ b/sound/soc/intel/skylake/skl-nhlt.c
@@ -160,11 +160,11 @@ struct nhlt_specific_cfg *
 
 	/* update the blob based on virtual bus_id*/
 	if (!skl->nhlt_override) {
-		dev_warn(bus->dev, "Querying NHLT blob from ACPI NHLT table !!\n");
+		dev_dbg(bus->dev, "Querying NHLT blob from ACPI NHLT table !!\n");
 		cfg = skl_get_ep_blob(skl, instance, link_type, s_fmt,
 				num_ch, s_rate, dir, dev_type);
 	} else {
-		dev_warn(bus->dev, "Querying NHLT blob from Debugfs!!\n");
+		dev_dbg(bus->dev, "Querying NHLT blob from Debugfs!!\n");
 		cfg = skl_nhlt_get_debugfs_blob(skl->debugfs, link_type, instance, dir);
 	}
 
-- 
1.9.1

